manifest.php

```diff
@@ -29,7 +29,7 @@
     'label' => 'LTI Delivery Tool Provider',
     'description' => 'The LTI Delivery Tool Provider allows third party applications to embed deliveries created in Tao',
     'license' => 'GPL-2.0',
-    'version' => '10.6.1',
+    'version' => '11.0.0',
```

Comment:
Was there a breaking change?

Reply:
Yes, we chaged interface of this public method - https://github.com/oat-sa/extension-tao-ltideliveryprovider/blob/44f9ce0bda2d54893fc5882a8d0577f83d415618/model/LTIDeliveryTool.php#L88